packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -1655,7 +1659,7 @@ class NotifierCheckbox extends StatelessWidget {
   Widget build(BuildContext context) {
     return ValueListenableBuilder(
       valueListenable: notifier,
-      builder: (context, value, _) {
+      builder: (context, dynamic value, _) {
```

Comment:
this should be bool. You may have to add the type to ValueListenableBuilder (`ValueListenableBuilder<bool>`)

Reply:
done